### ISSUE TITLE
games work with controller hotplug don't work anymore in 5.13

### ISSUE THREAD

**Issue by howdev** [ValveSoftware/Proton#4267]
Proton 5.13

Proton games that have controller hotplug working, now does not work anymore with 5.13

**Comment by DanMan** [comment]
Known issue: https://github.com/ValveSoftware/Proton/wiki/Changelog

**Comment by howdev** [comment]
Yes I realized in the 5.13 changelog, the hotplug is broken. It is not broken in the 5.09

**Comment by lhindir** [comment]
A bit off topic, but can you list some games with hotplugging functionality? I have many local multiplayer titles and have never found a game, native or Proton, that successfully reconnects my controllers, so I'd be interested to check one that someone else can verify works. I use wireless Steam Controllers, if it matters.

**Comment by howdev** [comment]
@lhindir hotplug was working on 4.11 and 5.09 with these games Darksiders 3, Borderlands 3, Remnant from the Ashes, Livelock. I use dualshock 4
